### ISSUE TITLE
Default logging of Bridge compiler is super-verbose

### ISSUE THREAD

**Issue by DanTup** [bridgedotnet/Bridge#3155]
We have a large number of Bridge projects and currently our Output window in VS (as well as logged output on the build server) is incredibly verbose.

I know this can be controlled in bridge.json, but it feels like it might be a bit noisey by default. The normal MSBuild output is fairly quiet, so it seems like Bridge should be similar by default (or ideally, use the switch already passed to msbuild).

**Comment by geoffreymcgill** [comment]
@DanTup – What setting for [logging](https://github.com/bridgedotnet/Bridge/wiki/global-configuration#logging) do you have configured in your **bridge.json**?

**Comment by DanTup** [comment]
@geoffreymcgill I don't believe we have any set anywhere. The output isn't as bad as when we've had it on trace in the past (for debugging), but it's orders of magnitude more chatty by default than typical msbuild output on a .net project.

**Comment by geoffreymcgill** [comment]
Thanks for pointing out this verbose logging. We have reviewed and agree that things can be tightened up. We should be able to implement these changes for the [16.4.0](https://github.com/bridgedotnet/Bridge/milestone/45) release.

**Comment by geoffreymcgill** [comment]
On a successful build, logging has been cleaned up and reduced to the following:

```
------ Rebuild All started: Project: Demo1, Configuration: Debug Any CPU ------
  Demo1 -> c:\users\user\Demo1\bin\Debug\Demo1.dll
  Bridge started
      To enable detailed logging, configure "logging" in bridge.json.
      https://github.com/bridgedotnet/Bridge/wiki/global-configuration#logging
  Bridge done
========== Rebuild All: 1 succeeded, 0 failed, 0 skipped ==========
````

You enable detailed logging by adding a [`logging`](https://github.com/bridgedotnet/Bridge/wiki/global-configuration#logging) config section to your projects **bridge.json**. 

#### Example

```json
{
  "logging": {
    "level": "Trace"
  },
}
````

Hope this helps.

**Comment by fabriciomurta** [comment]
The change mentioned above has been merged to bridge and will be available with 16.5.0 release.
